src/util/global_worker_pool.js

```diff
@@ -15,3 +15,24 @@ export default function getGlobalWorkerPool () {
     }
     return globalWorkerPool;
 }
+
+export function preloadWorkerPool() {
+    const workerPool = getGlobalWorkerPool();
+    workerPool.acquire(PRELOAD_POOL_ID);
+}
+
+export function removePreloadedWorkerPool() {
+    const pool = globalWorkerPool;
+    if (!pool) {
+        console.warn('Attempted to remove preloaded worker pool before creating one, skipping. Use mapboxggl.preloadWorkerPool() to preload a global worker pool');
```

Comment:
I don't think we should have a warning message here; this should just be a no-op.